test/circle/lint.sh

```diff
@@ -30,6 +44,28 @@ for directory in ${CHANGED_FOLDERS}; do
   if [ "$directory" == "incubator/common" ]; then
     continue
   elif [ -d $directory ]; then
-    helm lint ${directory}
+    run helm lint ${directory}
+
+    # If a Chart.yaml file is present lint it. Otherwise report an error
+    # because one should exist
+    if [ -e ${directory}/Chart.yaml ]
+    then
+        run yamllint -c test/circle/lintconf.yml ${directory}/Chart.yaml
```

Comment:
The rest of the file seems to be indented with two spaces. Can we agree on one style?